masappcli/elevenpaths_auth.py

```diff
@@ -114,7 +152,13 @@ def get_scan_result(self, workgroup, scan_id, scan_date, app_key, lang):
             raise ValueError(
                 "Language {language} Only supported languages: {langs}".format(language=lang, langs=LANGUAGES))
 
-        return self.get(url_path=self.API_SCAN_RESULT,
-                        authentication_instances=[self.authentication_instances],
-                        headers={'wg': workgroup},
-                        query_params={'scanId': scan_id, 'scanDate': scan_date, 'appKey': app_key, 'lang': lang})
+        if workgroup == None:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/app/alopezna5/mASAPP_CI/pullRequest?prid=4655639)